system/cu/cu_main.c

```diff
@@ -446,13 +446,15 @@ int main(int argc, FAR char *argv[])
 
   while (!cu->force_exit)
     {
-      int ch = getc(stdin);
+      int c = getc(stdin);
 
-      if (ch < 0)
+      if (c < 0)
         {
           continue;
         }
 
+      char ch = c;
```

Comment:
Looks good, but please move variable declaration to the beginning of the scope to respect C89 compatibility.
The other way is to eliminate `continue;` and put the whole code block under `if (c >= 0)` condition